src/py/flwr/cli/new/templates/app/code/server.xgboost.py.tpl

```diff
@@ -0,0 +1,54 @@
+"""$project_name: A Flower / XGBoost app."""
+
+from typing import Dict
+
+from flwr.common import Context
+from flwr.server import ServerApp, ServerAppComponents, ServerConfig
+from flwr.server.strategy import FedXgbBagging
+
+
+def evaluate_metrics_aggregation(eval_metrics):
+    """Return an aggregated metric (AUC) for evaluation."""
+    total_num = sum([num for num, _ in eval_metrics])
+    auc_aggregated = (
+        sum([metrics["AUC"] * num for num, metrics in eval_metrics]) / total_num
+    )
+    metrics_aggregated = {"AUC": auc_aggregated}
+    return metrics_aggregated
+
+
+def config_func(rnd: int) -> Dict[str, str]:
+    """Return a configuration with global epochs."""
+    config = {
+        "global_round": str(rnd),
+    }
+    return config
+
+
+def server_fn(context: Context):
+    # Read from config
+    num_rounds = int(context.run_config["num-server-rounds"])
+    pool_size = int(context.run_config["pool-size"])
```

Comment:
```suggestion
    pool_size = int(context.node_config["num-partitions"])
```

Reply:
@yan-gao-GY `pool_size` is the total number of clients that participate in the simulation right? If so i think it should be update with the above.